robottelo/cli/base.py

```diff
@@ -112,8 +138,8 @@ def create(cls, options=None):
             cls._construct_command(options), output_format='csv')
 
         # Extract new object ID if it was successfully created
-        if len(result.stdout) > 0 and 'id' in result.stdout[0]:
-            obj_id = result.stdout[0]['id']
+        if len(result) > 0 and 'id' in result[0]:
```

Comment:
it's been a bit since i've looked hard into our code but are we preferring ```if len(something) > 0``` over ```if something```? If so why? Thanks